docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+## Coordination and support procedures
+
+### Coordination Approach
+
+FIWARE Lab activities are managed by weekly scrum meetings and monthly
+Agile sprints. Every Tuesday at 15:00, all Nodes Administrators report
+their status and discuss different topics relating to the activities
+carried out within the Lab.
+
+The main topics covered within the meeting are:
+
+-   Status of the node
+
+    Each Node Administrator, provides a snapshot of the current status of
+    its node. He/She provides information about the presence within the
+    Infographic page (<http://infographic.lab.fiware.org>), about the
+    Sanity Checks status (<http://status.lab.fiware.org>), as well as
+    useful updates during specific activities like hardware maintenance,
+    users migration or OpenStack upgrade version.
+
+-   Issues occurred during the previous week
+
+    Each Node Administrator, discusses issues encountered during the past
+    week. This is a crucial point in order to identify and solve possible
+    weaknesses or bugs within the FIWARE Lab architecture (e.g. common
+    connectivity errors toward the centralized keystone) as well as
+    possible weakness in the FIWARE Lab documentation.
+
+-   Instructions from the FIWARE Lab Task Force or Technical Steering
+    Committee
+
+    This task is to inform all Nodes Administrators about decisions taken
+    by the FIWARE Lab Task Force or by the Technical Steering Committee
```

Comment:
by the FIWARE Technical Steering Committee and to design a roadmap of the future activities.